doc/.ChangeLog_template

```diff
@@ -29,6 +29,8 @@ Does this tag change answers significantly for any of the following physics conf
 
     [Put an [X] in the box for any configuration with significant answer changes.]
 
+[ ] clm5_1
```

Comment:
@ekluzek does it make sense to you that we should have this in here now, or did you deliberately exclude it? At some point soon we should also add a line for ctsm5_1-nwp, but I don't think we have any out of the box configurations like that yet.

Reply:
I actually purposefully excluded it -- because I figure clm5_1 is under development and will change in many tags, until we start a ctsm5_2 physics version. At that point I thought it would make sense to include it. But, we could go either way on this. I probably should have brought this up for discussion.

Reply:
Yeah, I also don't have strong opinions on this. For now I'll leave it, but I don't object to its removal. I think it comes down to whether it's helpful to the scientists to have this noted in the ChangeLog. Maybe we can discuss further at a ctsm-software meeting.